gr-soapy/grc/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright 2011 Free Software Foundation, Inc.
```

Comment:
I'm guessing that the original OOT started with an old version of GR, hence the old dates.  Should these be updated and made consistent going in-tree?